### PR TITLE
Brainnetome nibabel update

### PR DESCRIPTION
**Pull request by frheault**
Update of labels and image_math scripts for `get_fdata()` and `get_data_as_label` for a project related to freesurfer, brainnetome, glasser atlas.

This prevents warnings from nibabel in the scripts I made for Max.
(We have to start fixing these warning somewhere, this is a first pass)

### CODE REVIEW

File: scripts/scil_image_math.py [hunk 1]
```diff
@@ -60,17 +63,17 @@ def load_data(arg):
         data = float(arg)
     else:
         if not os.path.isfile(arg):
-            logging.error('Input file %s does not exist', arg)
-            raise ValueError
-        data = nib.load(arg).get_data()
-        logging.info('Loaded %s of shape %s and data_type %s',
-                     arg, data.shape, data.dtype)
+            raise ValueError('Input file {} does not exist.'.format(arg))
+        data = np.asanyarray(nib.load(arg).dataobj)
```
Comment (arnaudbore) [inline scripts/scil_image_math.py:67]: get_fdata ?

Comment (frheault) [reply scripts/scil_image_math.py:67]: Dans ce cas précis je veux avoir le type tel que déclarer dans le header et non le cast en float. Comme ça je peux ensuite détecter si y'a des types qui clash (et donc forcer mon --data_type)

File: scripts/scil_image_math.py [hunk 2]
```diff
@@ -37,17 +40,17 @@ def _build_arg_parser():
     p.add_argument('operation',
                    choices=OPERATIONS.keys(),
                    help='The type of operation to be performed on the '
-                   'images.')
-
+                        'images.')
     p.add_argument('inputs', nargs='+',
                    help='The list of image files or parameters.')
+    p.add_argument('output',
```
Comment (arnaudbore) [inline scripts/scil_image_math.py:46]: in_images

out_image

File: scripts/scil_split_volume_by_ids.py [hunk 3]
```diff
@@ -82,7 +83,7 @@ def main():
                      'Will not process.\nConvert your image to integers ' +
                      'before calling.')
 
-    label_image_data = label_image.get_fdata().astype(int)
+    label_image_data = get_data_as_label(label_image)
```
Comment (arnaudbore) [inline scripts/scil_split_volume_by_ids.py:86]: in_label instead of label_image

File: scripts/scil_split_volume_by_labels.py [hunk 4]
```diff
@@ -74,7 +75,7 @@ def main():
                      'Will not process.\nConvert your image to integers ' +
                      'before calling.')
 
-    label_image_data = label_image.get_fdata().astype(np.int)
+    label_image_data = get_data_as_label(label_image)
```
Comment (arnaudbore) [inline scripts/scil_split_volume_by_labels.py:78]: in_label
out_dir
out_prefix

File: scripts/scil_split_volume_by_ids.py [hunk 5]
```diff
@@ -73,53 +73,53 @@ def main():
...
 
-    required = args.label_image
+    required = args.in_labels
     assert_inputs_exist(parser, required)
 
-    label_image = nib.load(args.label_image)
+    label_img = nib.load(args.in_labels)
 
-    if not issubclass(label_image.get_data_dtype().type, np.integer):
+    if not issubclass(label_img.get_data_dtype().type, np.integer):
```
Comment (arnaudbore) [inline scripts/scil_split_volume_by_ids.py:81]: you can remove this, it's part of get_data_as_label

**Pull request merged**